timbermill-local/src/main/java/com/datorama/oss/timbermill/pipe/LocalOutputPipe.java

```diff
@@ -138,7 +138,7 @@ public DiskHandler getDiskHandler() {
         private String diskHandlerStrategy = "sqlite";
         private int maxFetchedBulksInOneTime = 100;
         private int maxInsertTries = 10;
-        private String locationInDisk = "/db";
+        private String locationInDisk = "/tmp";
         private String orphansAdoptionsCronExp = "0 0/1 * 1/1 * ? *";
```

Comment:
I think it should be a different location rather than "/db" in order to work locally, right?

Reply:
Not a bad idea